pkg/reporter/text.go

```diff
@@ -69,6 +69,9 @@ func init() {
 			"Red": func(items ...interface{}) string {
 				return color.New(color.FgRed).Sprint(items...)
 			},
+			"Link": func(items ...interface{}) string {
```

Comment:
Maybe this should be "URL" instead of "Link", that way "Link" is available for ["real" links](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda).

Also, after thinking about this a bit more, I think there is a usability problem here... the URLs aren't really clickable by default (at least in the emulators I've tried).  In iTerm2 I have to hold down option and click, in GNOME Terminal (and probably all VTE-based terminals, there are around a dozen IIRC) it's ctrl+click or right click and select  "Open Link".

Reply:
I think that behavior is expected. By usability issue, do you just mean that it's misleading to style them like hyperlinks?

Reply:
Yep.  I can definitely see people clicking on the "link" over and over only to have it not work and getting pretty frustrated.

Reply:
I don't mind us removing the underline fwiw. @jason-fugue I defer to you and what you want to do here.

Reply:
Yeah getting rid of the underline makes sense. It looks like at least the terminals I use add the underline already when you either mouse over or cmd+mouse over. I'll push that change.